### PR TITLE
Remove obsolete migrations

### PR DESCRIPTION
**Pull request by marc0der**
The migrations build became very unstable of late. This instability is mainly due to the URL validation performing a deep probe on each URL. Apache URLs don't seem to like being probed so frequently, causing them to fail periodically.

This PR removes _many_ old migrations that have zero value.

### CODE REVIEW

**Review by eddumelendez — APPROVED** [review]

**Comment by helpermethod** [comment]
I've left a few comments, though nothing fancy.

**Comment by marc0der** [comment]
@helpermethod I'm not seeing any comments, could you point me to them please?

**Comment by helpermethod** [comment]
https://github.com/sdkman/sdkman-db-migrations/pull/563#pullrequestreview-823426428

**Comment by marc0der** [comment]
@helpermethod super weird, but I see nothing here...

**Comment by helpermethod** [comment]
Ah sorry, the review was still pending.

File: src/main/scala/io/sdkman/changelogs/ScalaMigrations.scala [hunk 1]
```diff
@@ -5,378 +5,6 @@ import com.mongodb.client.MongoDatabase
...
-  )
-  def migration025(implicit db: MongoDatabase): Unit =
-    Version(
-      candidate = "scala",
-      version = "2.12.15",
-      url = "https://downloads.lightbend.com/scala/2.12.15/scala-2.12.15.zip"
-    ).validate()
-      .insert()
-
-  @ChangeSet(
```
[the remark this answers is not in the document]
Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/ScalaMigrations.scala:365]: I guess I got a bit carried away :laughing:

[the remark this answers is not in the document]
Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/ScalaMigrations.scala:365]: Actually just thinking, new versions of Scala 2 will be released by the GitHub actions changes that were merged into the Scala pipeline recently. All subsequent releases will happen via the Vendor API.

**Comment by marc0der** [comment]
GitHub pipelines are messed up :-(

It is running the build on master, not on the feature branch with all my deletions. Will need to take a closer look at the workflow at some point. Going to merge anyway.

**Pull request merged**